blog/2025/issue-credentials-over-openid4vci.adoc

```diff
@@ -0,0 +1,358 @@
+:title: Setting Up Keycloak as a Credential Issuer with OpenID4VCI
+:date: 2025-12-02
+:publish: true
+:author: Rodrick Awambeng, Forkim Enjeckayang, Ingrid Kamga, Bertrand Ogen
+:summary: Keycloak can act as a verifiable credential issuer using OpenID4VCI, allowing users to receive verifiable credentials as digital proofs of identity. This guide walks through the required configuration steps.
+
+Before configuring Keycloak, it is helpful to understand its role in decentralized identity ecosystems. As a verifiable credential issuer, Keycloak can issue digitally signed credentials using the https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html[OpenID for Verifiable Credential Issuance] (OpenID4VCI) protocol, allowing relying parties(also known as verifiers) to independently verify them without contacting the issuer.
+
+Keycloak implements OpenID4VCI, enabling the issuance of verifiable credentials (VCs) as digital proofs of identity or attributes. Configuring this functionality requires consistent setup across the realm, clients, and issuable credentials (client scopes).
+
+For example, consider the following scenario: The Keycloak OAuth SIG team wants to issue verifiable membership credentials to its members, including their name and email, which can later be presented at onsite or virtual events as proof of active membership.
+
+NOTE: At the time of writing this blog, Keycloak's support for OpenID4VCI is still experimental. This guide uses the https://www.keycloak.org/nightly/[**nightly release**] which builds from the main branch. The feature is expected to be promoted to preview in early 2026.
+
+== Introduction to OpenID4VCI
+
+OpenID4VCI is a protocol developed by the OpenID Foundation that extends the OAuth 2.0 framework to support the secure and interoperable issuance of Verifiable Credentials (VCs). VCs are digital, tamper-evident representations of information, such as identity attributes or qualifications, that can be cryptographically verified without needing to contact the issuer at verification time. This enables decentralized identity ecosystems in which users (holders) retain control of their data and can share it selectively with verifiers.
+
+.Triangle of trust or Issuer-Holder-Verifier model
+image::${blogImages}/openid4vci/triangle-of-trust.png[alt="Triangle of trust or Issuer-Holder-Verifier model", align=center, width=800]
```

Comment:
The image's title includes the word "holder" and "verifier", but the image itself does not include these words. How about changing the tile to tell what the image shows? Or adding "verifier" in the image. That might be preferable.